mne/io/brainvision/brainvision.py

```diff
@@ -224,6 +242,24 @@ def _synthesize_stim_channel(events, n_samp):
     return stim_channel
 
 
+def _check_hdr_version(header):
+    tags = ['Brain Vision Data Exchange Header File Version 1.0',
+            'Brain Vision Data Exchange Header File Version 2.0']
+    if not any([header.startswith(tag) for tag in tags]):
```

Comment:
Why startswith instead of `header in tags`? This code implies there is more text at the end of the line, is there? If so please add a comment.